docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -768,3 +768,55 @@ When I type text `pas1$`
 ----
 
 include::partial$generic-ui-steps.adoc[]
+
+=== Change network condition
+
+Turn ON/OFF network condition.
+
+TIP: For IOS devices: to return to the application  you must use a step:
+When I activate application with bundle identifier `$bundleId`.
+
+[source,gherkin]
+----
+When I turn '$toggle' '$mode' network connection
```

Comment:
Could you please accurately follow the proposed syntax: https://github.com/vividus-framework/vividus/pull/2108#discussion_r741774557 ?